.github/workflows/ci.yml

```diff
@@ -77,7 +77,7 @@ jobs:
         fetch-depth: 0
     - uses: actions/setup-python@v2
       with:
-        python-version: 3.8
+        python-version: '3.10'
```

Comment:
I figured it would be safe to bump everything to 3.10, but if there was a particular reason to keep things on 3.8 here let me know and I'll revert it.